internal/db/schema/migrations/oss/postgres/28/01_kms.up.sql

```diff
@@ -0,0 +1,194 @@
+begin;
+
+-- make the required schema changes to adopt
+-- github.com/hashicorp/go-kms-wrapping/extras/kms/v2 
+
+create table kms_schema_version(
+    version text not null,
+    create_time wt_timestamp,
+    update_time wt_timestamp
+);
+
+-- ensure that it's only ever one row
+create unique index kms_schema_version_one_row
+ON kms_schema_version((version is not null)); 
+
+insert into kms_schema_version(version) values('v0.0.1');
+
+-- we can use our existing kms_root_key table.
+
+-- we need to add an add additional constraint to the kms_roolt_key_version
+-- table: 
+-- https://github.com/hashicorp/go-kms-wrapping/blob/c06f9db9380f2c26cad07cf51f7a324d25dd55ba/extras/kms/migrations/postgres/04_keys.up.sql#L29 
+alter table kms_root_key_version
+  add constraint not_empty_key
+  check (
+      length(key) > 0
+    );
```

Comment:
Is this already the case today in all possible databases, or do we need to set/delete key_versions without a key first?

Reply:
key's should never be empty in existing databases.